Core/src/org/sleuthkit/autopsy/modules/hashdatabase/HashDbIngestModule.java

```diff
@@ -182,8 +184,20 @@ public ProcessResult process(AbstractFile file) {
         String md5Hash = file.getMd5Hash();
         if (md5Hash == null || md5Hash.isEmpty()) {
             try {
+                TimingMetric metric = EnterpriseHealthMonitor.getTimingMetric("Disk Reads: Hash calculation");
                 long calcstart = System.currentTimeMillis();
                 md5Hash = HashUtility.calculateMd5Hash(file);
+                if (file.getSize() > 0) {
+                    // Surprisingly, the hash calculation does not seem to be correlated that
+                    // strongly with file size until the files get large.
+                    // Only normalize if the file size is greater than ~1MB.
+                    if (file.getSize() < 1000000) {
+                        EnterpriseHealthMonitor.submitTimingMetric(metric);
+                    } else {
+                        // In testing, this normalization gave reasonable resuls
+                        EnterpriseHealthMonitor.submitNormalizedTimingMetric(metric, file.getSize() / 500000);
```

Comment:
I would prefer a named constant here, instead of a "magic number." Also note typo in comment.